### PR TITLE
Enable multi-window support in the manifest.

### LINKED ISSUE #2594: Please support Multi-Window mode on Android 7.0 +
### Expected behavior

When in k-9 mail, and performing a long press on the Overview button, then selecting a second application to appear next to k-9 mail (such as Chrome) should result in having k-9 mail and that application side by side with no errors or warnings. 

### Actual behavior

When in k-9 mail, and performing a long press on the Overview button, when the screen splits and the option to select the second applcation appears on the right (as expected), there is a warning that appears on k-9 mail "App may not work with split-screen". 

### Steps to reproduce
1. In k-9 mail, built from master (72af527db083327e871d9ef80b02c31ac3b799f1), running on Android 7.0 or newer, perform a long press on the Overview button.
2. See warning appear on k-9 mail.

### Environment
K-9 Mail version: 5.207, also seen in master (72af527db083327e871d9ef80b02c31ac3b799f1)

Android version:7.0 on Nvidia Shield K1, SHIELD tablet software version 5.2 (24.29.303.153)

Account type (IMAP, POP3, WebDAV/Exchange): IMAP

### PR DESCRIPTION
**Pull request by jd-boyd**
This is intended to resolve #2594.

When I run the unit tests locally, the before and after results are the same.  As this is a manifest change only, I don't think that a new unit test is needed, but I would be happy to accept guidance here.

Thank you for your time.

### CODE REVIEW

**Comment by philipwhiuk** [comment]
Does this actually work on a device and not just throw up the Toast saying 'might not work'? 

The documentation suggestions that you need to target API 24 or higher which we don't because of the Runtime Permissions thing. See #781: 

> **If your app targets API level 24 or higher**, you can configure how and whether your app's activities support multi-window display. 

(emphasis mine)

https://developer.android.com/guide/topics/ui/multi-window.html

**Comment by jd-boyd** [comment]
> Does this actually work on a device and not just throw up the Toast saying 'might not work'?

This is actually working on my device (Nvidia Shield K1).  The build.gradle file locally still says

        minSdkVersion 15
        targetSdkVersion 22

The generated manifests that appear in the k9mail/build directory also keep saying 22.  We can close this as me being an idiot, but it would be nice to know what I could be doing that makes it appear to work if it shouldn't be.

BTW, one thing I found: https://developer.android.com/reference/android/R.attr.html#resizeableActivity

    The default value is false for applications with targetSdkVersion lesser than N and true otherwise.

Perhaps that means that applications with targetSdkVersion lesser than N can still override the default value?  That made me wonder what happen if I tried deploying this change to an Android 5.1.1 device. I don't have one so I used the emulator set to Nexus 7 API 22, and this also doesn't seem to cause any trouble there.

**Comment by philipwhiuk** [comment]
Hmm that documentation seems to contradict the official docs. But if it works I'm happy to see it merged.

File: k9mail/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -65,6 +65,8 @@
         android:icon="@drawable/icon"
         android:label="@string/app_name"
         android:theme="@style/Theme.K9.Startup"
+        android:resizeableActivity="true"
+        android:supportsPictureInPicture="true"
```
Comment (cketti) [inline k9mail/src/main/AndroidManifest.xml]: According to the documentation `supportsPictureInPicture` needs to be specified on an Activity level. However, K-9 Mail neither supports picture in picture mode nor is it a video player app. So we don't want to set this attribute in our manifest.

**Comment by jd-boyd** [comment]
@cketti I have removed that line.  Thanks for pointing that out.

**Comment by cketti** [comment]
Thanks for the pull request.

**Comment by jd-boyd** [comment]
> Thanks for the pull request.

I don't think you actually merged that second commit.  Do you need that in an additional pull request?

**Comment by cketti** [comment]
I combined the two commits into one. That way the history of `master` won't include unnecessary changes.

**Pull request closed without merging**